### PR TITLE
Update JWT packages and code

### LINKED ISSUE #585: Update pyjwt to the latest version
pyjwt's latest 2.0 release has had some breaking changes, and our app needs to be updated to work with it.
See codeforpdx/dwellinglybackend/pull/240 for more information.

### PR DESCRIPTION
**Pull request by aedwardg**
### What issue is this solving?

Closes codeforpdx/dwellingly-app/issues/585

Initially I was just going to update PyJWT, but it wouldn't work without also updating Flask-JWT-Extended.
Both packages had a significant amount of breaking changes, but everything should be working correctly after this.

For more information on the changes to both packages, see:
- https://pyjwt.readthedocs.io/en/stable/changelog.html#v2-0-0
- https://flask-jwt-extended.readthedocs.io/en/stable/v4_upgrade_guide/

### Any helpful knowledge/context for the reviewer?
Is a re-seeding of the database necessary? NO
Any new dependencies to install? YES
Any special requirements to test? Not sure, I don't think so. All tests are passing on my end.

### Please make sure you've attempted to meet the following coding standards
- [x] Code builds successfully
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -98,8 +98,8 @@ def role_loader(identity):
 
     # checking if the token's jti (jwt id) is in the set of revoked tokens
     # this check is applied globally (to all routes that require jwt)
-    @app.jwt.token_in_blacklist_loader
-    def check_if_token_in_blacklist(decrypted_token):
+    @app.jwt.token_in_blocklist_loader
+    def check_if_token_in_blacklist(jwt_header, decrypted_token):
```
Comment (NickSchimek) [inline app.py]: Is this correct? Should the a be an o?

**Review by NickSchimek — COMMENTED** [review]
Thank you. I think there is one typo

File: app.py (hunk 1 above)
Comment (aedwardg) [reply app.py]: Sorry, that was the old function name and I left it the same when I updated the decorator.
Changed it now.

**Review by NickSchimek — APPROVED** [review]
Awesome, thanks for doing this!

**Pull request merged**